go/interpreter/lfvm/memory.go

```diff
@@ -237,48 +228,22 @@ func (m *Memory) CopyData(offset uint64, trg []byte) {
 	}
 }
 
-func (m *Memory) GetSlice(offset, size uint64) []byte {
+func (m *Memory) getSlice(offset, size uint64) []byte {
```

Comment:
Please document what should happen with out-of-bound data requests and add tests for that of there are none yet.